reference/features/search_parameters.md

```diff
@@ -299,25 +299,27 @@ If `attributesToCrop` is not configured, `cropLength` has no effect on the retur
 **Expected value**: an array of <clientGlossary word="attribute" label="attributes" /> or `["*"]`
 **Default value**: `null`
 
-Highlights matching query terms in the given attributes. When this parameter is set, the `_formatted` object is added to the response for each document, within which you can find the highlighted text.
+Highlights matching query terms in the specified attributes by enclosing them in `<em>` tags. `attributesToHighlight` highlights matches in fields containing strings, numeric values, arrays, and objects.
 
-Values can be supplied as an array of attributes: `attributesToHighlight=["attributeA", "attributeB"]`.
+When this parameter is set, returned documents include a `_formatted` object containing the highlighted terms.
 
-Alternatively, you can provide `["*"]` as a value: `attributesToHighlight=["*"]`. In this case, all the attributes present in `attributesToRetrieve` will be assigned to `attributesToHighlight`.
+Parameter values must be given as an array of attributes: `attributesToHighlight=["attributeA", "attributeB"]`.
```

Comment:
Is it necessary to include this information given the "Expected value" field above?

Reply:
Not strictly necessary, no, but perhaps useful anyhow because we present an example? Don't feel strongly about it either way—if you think it's best to remove it, I'm happy to do it.